devices/Hdc1080/BitHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Hdc1080
+{
+    /// <summary>
+    /// Helper that allows setting and getting each bit separately
+    /// </summary>
+    public static class BitHelper
+    {
+        public static byte SetBit(this byte b, int pos, bool value)
+        {
+            if (value)
+                return (byte)(b | (1 << pos));
```

Comment:
We're always using brackets in if instruction to reduce issues. So please use { and } for all the if instructions.

Reply:
Fixed